js-arrays-objects-loops/objects-homework/objects.js

```diff
@@ -0,0 +1,18 @@
+//1.
+const userProfile = {
+    userName: 'Sara',
+    lastName: 'Mojsova',
+    years: 23,
+    idNumber: 368,
+    hasDrivingLicense: true,
+    emails: ['saramojs@gmail.com', 'sara-mojsova@live.com', 'sara-mojsova@hotmail.com'],
+
+    //3.
+    printData: function () {
+        console.log(`This user\'s name is ${this.userName} and she is ${this.years} years old. She has a driving license and she is adult.`);
+    }
```

Comment:
We need some condition inside to render if the user has driving license and if the user is an adult. You can see in the example below.

console.log(`This user's name is ${this.userName} and he is ${this.years} years old. He has ${this.hasDrivingLicense ? 'a' : 'no'} driving license and he ${this.years > 18 ? 'is' : 'is no'} adult`)